internal/clients/team2/client.go

```diff
@@ -184,7 +184,7 @@ func (c *client) gameConfig() config.ClientConfig {
 // You will need it to access the game state through its GetGameStateMethod.
 func (c *client) Initialise(serverReadHandle baseclient.ServerReadHandle) {
 	c.ServerReadHandle = serverReadHandle
-	c.LocalVariableCache = rules.CopyVariableMap()
+	c.LocalVariableCache = rules.CopyVariableMap(c.ServerReadHandle.GetGameState().RulesInfo.VariableMap)
```

Comment:
make sure to confirm with team 2 when changing this code

Reply:
Team 2 approves

Reply:
Cheers